addons/web/static/tests/views/kanban_tests.js

```diff
@@ -3728,6 +3728,61 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('nocontent helper after adding a record(kanban with progressbar)', async function (assert) {
+        assert.expect(3);
+
+        const kanban = await createView({
+            View: KanbanView,
+            model: 'partner',
+            data: this.data,
+            arch: `<kanban >
+                    <field name="product_id"/>
+                    <progressbar field="foo" colors='{"yop": "success", "gnap": "warning", "blip": "danger"}' sum_field="int_field"/>
+                    <templates>
+                      <t t-name="kanban-box">
+                        <div><field name="foo"/></div>
+                      </t>
+                    </templates>
+                </kanban>`,
+            groupBy: ['product_id'],
+            domain: [['foo', '=', 'abcd']],
+            mockRPC: function (route, args) {
+                if (args.method === 'web_read_group') {
+                    const result = {
+                        groups: [
+                            { __domain: [['product_id', '=', 3]], product_id_count: 0, product_id: [3, 'hello'] },
+                        ],
+                    };
+                    return Promise.resolve(result);
+                }
+                return this._super.apply(this, arguments);
+            },
+            viewOptions: {
+                action: {
+                    help: "No content helper",
+                },
+            },
+        });
+
+        assert.strictEqual(kanban.$('.o_view_nocontent').length, 1,
```

Comment:
use containsOnce/containsNone

Reply:
sorry, I copied/pasted code from the 12.0 PR